pmm-app/tests/pages/pmmInventoryPage.js

```diff
@@ -207,4 +208,13 @@ module.exports = {
 
     assert.equal(count, otherDetails, 'Check data!');
   },
+
+  async getCellValue(index, column) {
```

Comment:
could we have better naming which makes it understandable for fresh eyes, please? 

maybe `row` `column`? or better we could just ask user to input Row Number and Column Name ?